### PR TITLE
Updates to InstructorSendEmail help.

### PR DESCRIPTION
**Pull request by somiaj**
These updates match the changes in #1986.

### CODE REVIEW

File: templates/HelpFiles/InstructorSendMail.html.ep [hunk 1]
```diff
@@ -17,19 +17,43 @@
 % title maketext('Email Help');
 %
 <p>
-	<%= maketext('Emails can be sent from this page to the class. Be sure to change all blanks which read "fixme". The '
-		. 'email can be sent to all students, or some subset. To change the way the students are ordered, after '
-		. 'selecting the criteria, push the "Change Display Settings" button.') =%>
+	<%= maketext('Use this page to send emails to active (enrolled or auditing) students.  Emails can be sent to '
+		. 'all active students or selected students.  Use the "Students" form to sort, filter, and/or format how '
```
Comment (drgrice1) [inline templates/HelpFiles/InstructorSendMail.html.ep]: Please change "and/or" to just "or".  Instructors of courses that use WeBWorK (mostly mathemeticians) know that "or" can be inclusive.

**Review by drgrice1 — APPROVED** [review]
This looks good.  I will merge it as soon as openwebwork/webwork2#1986 is merged.

File: templates/HelpFiles/InstructorSendMail.html.ep (hunk 1 above)
Comment (somiaj) [reply templates/HelpFiles/InstructorSendMail.html.ep]: Done. Thanks.

**Comment by drgrice1** [comment]
Thanks for making the change.

**Pull request merged**